### PR TITLE
Updated 201-logic-app-transform-function template: Perf optimizations

### PR DESCRIPTION
**Pull request by jaakhter**
### Contributing guide
https://github.com/Azure/azure-quickstart-templates/blob/master/1-CONTRIBUTION-GUIDE/README.md

### Changelog
Updated 201-logic-app-transform-function template
*
*
*
*

### Description of the change
The updated template file contains the modified source code of Transform Azure function with Perf optimizations.

### CODE REVIEW

**Comment by azurecla** [comment]
Hi __@jaakhter__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <span>
        In order for us to evaluate and accept your PR, we ask that you sign a contribution license agreement. It's all electronic and will take just minutes. I promise there's no faxing. https://cla.azure.com.
    </span>

TTYL, AZPRBOT;

**Comment by jaakhter** [comment]
The template requires a pre-existing Azure function app for deployment. Hence, the Travis CI build will fail and can be ignored.

**Comment by jaakhter** [comment]
@bmoore-msft, Can you please look into my request? It's a fix needed by customers by today.

File: 201-logic-app-transform-function/metadata.json [hunk 1]
```diff
@@ -2,6 +2,6 @@
     "itemDisplayName": "Azure function with transform capabilites",
     "description": "Creates a webhook based C# azure function with transform capabilites to use in logic apps integration scenarios",
     "summary": "Creates a webhook based C# azure function with transform capabilites",
-    "githubUsername": "rarayudu",
-    "dateUpdated": "2016-06-02"
+    "githubUsername": "jaakhter",
```
Comment (bmoore-msft) [inline 201-logic-app-transform-function/metadata.json]: only change this if you want to own the template going fwd..

**Comment by bmoore-msft** [comment]
@jaakhter - re: the pre-existing fn app - how does one get the sample to deploy then?  I.e. if someone wants to take & use the sample, how will they get it to work?

**Comment by jaakhter** [comment]
@bmoore-msft, There is existing documentation on this. Here is the link:
https://azure.microsoft.com/en-us/documentation/articles/app-service-logic-enterprise-integration-transform/

It talks about creating the function app as a prerequisite. 

Customers have already followed and created their apps. This new update is just some perf issue fixes reported by few customers.

**Comment by jaakhter** [comment]
@bmoore-msft, just some clarifications here. The prerequisite is to create an empty function app container. The function app is actually created by the template. The template requires the name of the existing function app container as input. Sorry for the confusion.

File: 201-logic-app-transform-function/metadata.json (hunk 1 above)
Comment (jaakhter) [inline 201-logic-app-transform-function/metadata.json]: Yes, I'll be owning this template for now. There are more work to be done by me on this. Hence, changed it,

**Comment by bmoore-msft** [comment]
Ok, I grabbed the update - when you get a chance can we add another template (or script) that will create the function app - so we have everything needed in the sample?

**Comment by jaakhter** [comment]
Thank you for the merge.

Regarding the template for creating the function app, its a good suggestion. However, we are working on a solution where this function app will be owned by us and there will not be any need for the customers to create or maintain it. So, even the template which we have is going to be removed,

**Pull request merged**